src/mscorlib/src/System/String.Comparison.cs

```diff
@@ -1144,5 +1144,16 @@ public override int GetHashCode()
 
             return referenceCulture.CompareInfo.IsPrefix(this, value, ignoreCase ? CompareOptions.IgnoreCase : CompareOptions.None);
         }
+
+        [Pure]
+        public Boolean StartsWith(Char value)
```

Comment:
Actually, it may be faster to check `m_firstChar` directly, e.g.:

```c#
public bool StartsWith(char value) => Length != 0 && m_firstChar == value;
```